Update Managing Content guide (SCA, Activation keys) (#1999)
theforeman/foreman-documentation: guides/common/modules/snip_only-valid-if-sca-disabled.adoc

[IMPORTANT]
====
This procedure is only valid if you have Simple Content Access (SCA) disabled on your {Project}.

> If you have SCA enabled, subscriptions are automatically attached to your host.

Reviewer: ```suggestion
If you have SCA enabled, having subscriptions attached to your hosts is not needed.
```
